user-doc/annotation.md

```diff
@@ -418,37 +418,25 @@ Supported JSON payload patterns:
+- `{`... `"name": ` _pattern_ ...`}`: A name for the asset to be derived by a [Pattern Expansion](#pattern-expansion) on _pattern_. The name MAY be used to represent a file name but is not required to do so.
+- `{`... `"bytes": ` _column_ ...`}`: The _column_ stores the file size in bytes of the asset. It SHOULD be an integer typed column.
+- `{`... `"sha256": ` _column_ ...`}`: The _column_ stores the checksum generated by the 'sha256' cryptographic hash function. It MUST be ASCII/UTF-8 hexadecimal encoded.
 - `{`... `"creator": ` _column_ ...`}`: The _column_ stores the _creator_ of the asset. The _column_ MAY be defined as a foreign key that references a person or user table in the model. The usual model introspection applies.
 - `{`... `"description": ` _pattern_ ...`}`: A description for the asset to be derived by a [Pattern Expansion](#pattern-expansion) on _pattern_.
-- `{`... `"date": ` _column_ ...`}`: The _column_ stores the date that the asset metadata was recorded in the system.
-- `{`... `"mediatype": ` _column_ ...`}`: The _column_ stores the Media Type of the asset, for example `application/dicom`. The format of its contents MUST conform to the [Media Type RFC](https://tools.ietf.org/html/rfc6838). The values used SHOULD come from the [list of Media Types](http://www.iana.org/assignments/media-types/media-types.xhtml) registered with IANA.
-- `{`... `"format": ` _column_ ...`}`: The _column_ stores the identifier for the file format that the asset _conforms to_, for example `http://edamontology.org/format_3548`. See the [EDAM Ontology](http://edamontology.org) for a listing of file formats used in bioinformatics.
-- `{`... `"contributor": ` _column_ ...`}`: The _column_ stores the "contributors" of the asset. The concept of contributor is not strictly defined here and left up to the application to define the exact criteria for what constitutes a contributor. When this key is present in the JSON payload the _column_ should be a JSON list of contributors. If this key is not present, applications should look for an inbound foreign key reference with a `from_name` annotation set to `'contributor'`, or the application should look for a relationship via a binary association table with a `to_name` annotation set to `'contributor'`. The entities of that related table should be interpreted as the set of contributors to the asset.
-
-#### Required Properties
-
-The following table specifies the required properties for the annotation. We also recommend additional properties for enhanced functionality provided by some client tooling.
-
-| Property     | Required | Recommended |
-|:-------------|:--------:|:-----------:|
-| `identifier` | X | - |
-| `mediatype`  | X | - |
-| `bytes`      | - | X |
-| `sha256`     | - | X |
-| `format`     | - | X |
-| `name`       | - | X |
+- `{`... `"date": ` _column_ ...`}`: The _column_ stores the date that the asset metadata was recorded in the system. The date MUST be in ISO 8601 format.
```

Comment:
I think we should require the column to be of a date or timestamp type, rather than talking about encoding.